src/solvers/dgmulti/flux_differencing_gauss_sbp.jl

```diff
@@ -25,22 +25,29 @@ function tensor_product_quadrature(element_type::Hex, r1D, w1D)
   return rq, sq, tq, wq
 end
 
-# type parameters for TensorProductFaceOperator
+# type parameters for `TensorProductFaceOperator`.
 abstract type AbstractGaussOperator end
 struct Interpolation <: AbstractGaussOperator end
-struct Projection  <: AbstractGaussOperator end
-
-#     TensorProductGaussFaceOperator{Tmat, Ti}
-# 
+# - `Projection{ScaleByFaceWeights=Static.False}` corresponds to the operator `Pf = M \ Vf'`,
+#   which is used in `VolumeIntegralFluxDifferencing`.
+# - `Projection{ScaleByFaceWeights=Static.True}` corresponds to the quadrature-based lifting
```

Comment:
```suggestion
# - `Projection{ScaleByFaceWeights=Static.False()}` corresponds to the operator `Pf = M \ Vf'`,
#   which is used in `VolumeIntegralFluxDifferencing`.
# - `Projection{ScaleByFaceWeights=Static.True()}` corresponds to the quadrature-based lifting
```
It's usually better to work with instances instead of types.